job-server/src/main/resources/html/js/spark-jobserver-ui.js

```diff
@@ -10,7 +10,7 @@ function getJobs() {
       $.each(jobs, function(key, job) {
         var items = [];
         items.push("<tr>");
-        items.push("<td>" + job.jobId + "</td>");
+        items.push("<td><a href='./jobs/" + job.jobId + "'>" + job.jobId + "</a> (<a href='./jobs/" + job.jobId + "/config'>C</a>)</td>");
```

Comment:
I defaulted to `C` since a wall of `Config` seemed overwhelming to me. But I can change it, please let me know either way.

![screen shot 2016-01-15 at 8 42 16 am](https://cloud.githubusercontent.com/assets/181554/12428457/78d9c990-be99-11e5-981d-6c059471cc17.png)